templates/magazine/_federated_info.html.twig

```diff
@@ -1,6 +1,24 @@
-{% if magazine.apId %}
-    <div class="alert alert__info">
-        <p>{{ 'federated_magazine_info'|trans }} <a
-                    href="{{ magazine.apProfileId }}">{{ 'go_to_original_instance'|trans }}</a></p>
-    </div>
+{% if magazine.apId %}   {# I.e. if we're federated #}
+    {% if entries is defined and entries and not entries.hasNextPage %}
+        {# Then show a link to original if we're at the end of content #}
+        <div class="alert alert__info">
+            <p>{{ 'federated_magazine_info'|trans }} <a
+             href="{{ magazine.apProfileId }}">{{ 'go_to_original_instance'|trans }}</a></p>
+        </div>
+    {% endif %}
+
+    {% if not magazine_has_local_subscribers(magazine) %}
+        {# Also show a warning if we're not actively receiving updates #}
+        {% set fetchedAt = magazine.apFetchedAt|date('U') %}
```

Comment:
The fetch date is not really the best thing to check for the last updated timestamp. The `apFetchedAt` is the last date the actor was updated, so the title, describtion, etc. The better thing to check would be the last created entry, entrycomment, post or postcomment. As this actually is a "last received update".
For example the `apFetchedAt` date will also get updated if a user searches for this magazine.

Reply:
Yeah, makes sense. That was why I worded it in terms of the last attempt instead of last update, but yeah, last real update is better. I'll change it accordingly.

Question though: If someone local creates a post or comment on a federated-but-disconnected magazine, won't that create content that would get counted as within the magazine? I know Lemmy works that way. If mbin does the same, maybe it's better to check for content that's associated with the magazine but has an .apId field (or whatever) set, so we know it was an actual received-from-the-origin update.

Reply:
Hm... how about this? This proposal is using magazine.lastActive. So, on my instance, I actually do get federated stuff coming into even magazines that are disconnected from their home instances (I'm using hexbear to test, since I'm not on their whitelist). So even iterating over entities associated with a magazine and testing their .apId values won't work.

.lastActive seems like it does get updated by successful communication with the magazine's home instance, and it seems like checking over my server it has a roughly-appropriate value for disconnected magazines. I don't understand the issues in full, but it looks like it's easy and simple to check, and equally as accurate as anything more complex option I can see. What do you think?

Reply:
I have to look when this value is actually set

Reply:
Sounds good. Yeah, I switched to it purely based on looking at some common cases on my instance without really understanding much behind it.

Reply:
I've done the groundwork for this in #503 
When that is merged, you can just change from `lastActive` -> `lastOriginUpdate` :)

Reply:
Done